internal/e2e/rollup/BUILD.bazel

```diff
@@ -1,17 +1,6 @@
 load("@build_bazel_rules_nodejs//:defs.bzl", "jasmine_node_test")
 load("//:defs.bzl", "rollup_bundle")
 
-# The local package node_modules filegroup contains just one depenedency to test
-# importing from a non-WORKSPACE-root node_modules folder. This makes it more clear
```

Comment:
are we losing some coverage of the coarse-grained deps?

Reply:
I don't think so in this case. That comment was from before fine grained deps existed and basically says that the rollup related deps don't need to be in the user's node_modules which is still the case in this test with fine grained dep. I updated it to just the depend on the minimum `@npm//hello` which is a better test.